lib/cors-anywhere.js

```diff
@@ -219,7 +219,8 @@ var getHandler = exports.getHandler = function(options, proxy) {
     originWhitelist: [],      // If non-empty, requests not from an origin in this list will be blocked.
     requireHeader: null,      // Require a header to be set?
     removeHeaders: [],        // Strip these request headers.
-    setHeaders: {}            // Set these request headers.
+    setHeaders: {},            // Set these request headers.
```

Comment:
When you resolve the merge conflict (and lint error), please make sure that the comments are visually aligned.